akka-docs/src/main/paradox/typed/persistence.md

```diff
@@ -248,6 +248,35 @@ Java
 Any `SideEffect`s are executed on an at-once basis and will not be executed if the persist fails.
 The `SideEffect`s are executed sequentially, it is not possible to execute `SideEffect`s in parallel.
 
+## Replies
+
+The @ref:[Request-Response interaction pattern](interaction-patterns.md#request-response) is very common for
+persistent actors, because you typically want to know if the command was rejected due to validation errors and
+when accepted you want a confirmation when the events have been successfully stored.
+
+Therefore you typically include a @scala[`ActorRef[ReplyMessageType]`]@java[`ActorRef<ReplyMessageType>`] in the
+commands. After validation errors or `thenRun` side effect the reply message can be sent to the `ActorRef`.
+
+TODO example of thenRun reply
```

Comment:
I want to use the examples from #25779 and therefore leaving this as a TODO, since that PR is on top of this PR.